chore: a start at having explanations for every contract (#386)
Agoric/documentation: main/zoe/guide/contracts/sell-items.md

# Sell Items

<Zoe-Version/>

##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/src/contracts/sellItems.js)
##### [View all contracts on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts)

Sell items in exchange for money. Items may be fungible or
non-fungible and multiple items may be bought at once. Money must be
fungible.

> The `pricePerItem` is to be set in the terms. It is expected that all
> items are sold for the same uniform price.

Reviewer: ```suggestion
The `pricePerItem` is set in the terms. It is expected that all
items sell for the same uniform price.
```

The initial offer should be `{ give: { Items: items } }`, accompanied by
terms as described above. Buyers use offers that match `{ want: {
Items: items } give: { Money: m } }`. The items provided should match
particular items that the seller still has available to sell, and the
money should be pricePerItem times the number of items requested.

> When all the items have been sold, the contract will terminate,

Reviewer: ```suggestion
When all the items have sold, the contract will terminate,
```

triggering the creator's payout. If the creator has an onDemand exit

> clause, they can exit early to collect their winnings. The remaining

Reviewer 2: ```suggestion
clause, they can exit early to collect their earnings. The remaining
```
